### PR TITLE
Resolve inline imports REVIEW __init__ CHANGES FIRST

### PR DESCRIPTION
**Pull request by fall-byrd**
## TL;DR

!! BEFORE REVIEWING THIS REVIEW https://github.com/FourThievesVinegar/solderless-microlab/pull/266 !!

This PR contains the changes for the aforementioned PR and moves imports out of inline to top level. Some additional small changes/fixes as well.

## What
What is affected by this PR?
- [ ] API
- [ ] GUI
- [ ] Hardware Integration
- [ ] OS/Deployment
- [ ] Documentation
- [X] Other (please describe in notes)

## Testing
How was this tested?
- [ ] Locally Virtualized
- [ ] Raspberry Pi
- [ ] With Hardware
- [X] Other (please describe in notes)

## Notes
Did the usual testing: unit tests, ran the server, poked around in the UI with the server running.

### CODE REVIEW

**Comment by fall-byrd** [comment]
Ran into an exception in the CI tests when attempting to load the `w1thermsensor` library however didn't see that locally testing though I don't have the sensor installed locally either. Odd.

**Comment by fall-byrd** [comment]
Github doesn't seem to have updated this PR to deal with the fact that the __init__ move was merged even with a branch update. Going to open a new PR of this again.
